src/__generated__/ArtworkSidebarPartnerInfo_artwork.graphql.ts

```diff
@@ -4,7 +4,6 @@ import { ConcreteFragment } from "relay-runtime";
 declare const _ArtworkSidebarPartnerInfo_artwork$ref: unique symbol;
 export type ArtworkSidebarPartnerInfo_artwork$ref = typeof _ArtworkSidebarPartnerInfo_artwork$ref;
 export type ArtworkSidebarPartnerInfo_artwork = {
-    readonly collecting_institution: string | null;
```

Comment:
I'm not sure why you have updates to these files?

Reply:
Not sure, just pulled in latest from master: 

<img width="443" alt="screen shot 2018-11-20 at 2 41 23 pm" src="https://user-images.githubusercontent.com/236943/48807444-62d1bd00-ecd2-11e8-8c32-b9179acf604f.png">

Reply:
<img width="285" alt="screen shot 2018-11-20 at 2 42 02 pm" src="https://user-images.githubusercontent.com/236943/48807471-7715ba00-ecd2-11e8-869a-7400afea8171.png">